src/ceph-volume/ceph_volume/util/__init__.py

```diff
@@ -2,6 +2,13 @@
 from math import floor
 from ceph_volume import terminal
 
+try:
+    # Map raw_input() to input() in Python2
+    import __builtin__
+    input = getattr(__builtin__, 'raw_input')  # pylint: disable=redefined-builtin
```

Comment:
there is no need for a `getattr()` call here, this could simply set  `input = raw_input` and try/except the `NameError`:

    try:
        input = raw_input
    except NameError:
        pass